adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/Pagination.kt

```diff
@@ -0,0 +1,23 @@
+package net.adoptopenjdk.api.v3
+
+import kotlin.math.min
+
+object Pagination {
+    private const val defaultPageSizeNum = 10
+    private const val maxPageSizeNum = 500
+    const val defaultPageSize = defaultPageSizeNum.toString()
+    const val maxPageSize = maxPageSizeNum.toString()
+
+    fun <T> getPage(pageSize: Int?, page: Int?, releases: Sequence<T>): List<T>? {
+        val pageSizeNum = min(maxPageSizeNum, (pageSize ?: defaultPageSizeNum))
+        val pageNum = page ?: 0
+
+        val chunked = releases.chunked(pageSizeNum)
+
+        return try {
+            chunked.elementAt(pageNum)
+        } catch (e: IndexOutOfBoundsException) {
+            null
```

Comment:
I think it may be better to throw the exception here rather than return null then throw

Reply:
✅